### PR TITLE
Update Readme.rst

### PR DESCRIPTION
**Pull request by N1ghtCrawl3r**
Added installation guide and usage with some RST formatting

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,6 +36,72 @@ Pick n Mix is a simple stacking tool for stacking Sci-Kit learn models of your p
...
+You can also use preprocessing in a Layer:
+
+::
+
+    from sklearn.linear_model import MinMaxScaler
+
+    first_layer = Layer([LinearRegression(), Ridge()],
+                        preprocessors = [MinMaxScaler(), None])
+                        
+For more examples for usage, please refer to the _`documentation < https://picknmix.readthedocs.io>`_.
```
Comment (Cheukting) [inline README.rst]: I think the _ beofre ` is not needed

**Comment by Cheukting** [comment]
Thank you @N1ghtCrawl3r , only a small change required. It looks very detail and professional. Also, do you want to put your name in the AUTHORS.rst?

File: README.rst (hunk 1 above)
Comment (N1ghtCrawl3r) [reply README.rst]: Okay, I'll correct it now. Thank you for your suggestion.

**Comment by Cheukting** [comment]
@N1ghtCrawl3r It looks perfect! You sure don't want to add your name in AUTHORS.rst? All contributions count.

**Comment by N1ghtCrawl3r** [comment]
Yes, I'll be glad if you could add my name.

**Comment by Cheukting** [comment]
@N1ghtCrawl3r ok, I will merge this PR now. Feel free to send another PR to add your name in AUTHORS.rst or I can do it for you next time I open a PR.

**Comment by N1ghtCrawl3r** [comment]
Okay thank you. You can add my name in the authors' list.

**Pull request merged**